experimental/dds/tree2/src/test/feature-libraries/typedSchema/flexList.spec.ts

```diff
@@ -76,17 +78,25 @@ import { requireAssignableTo, requireFalse, requireTrue } from "../../../util";
 }
 
 describe("FlexList", () => {
-	it("normalizeFlexList", () => {
+	it("correctly normalizes lists to be lazy", () => {
 		const list = [2, (): 1 => 1] as const;
-		const normalized = normalizeFlexList(list);
+		const normalized = normalizeFlexListLazy(list);
 		assert(normalized.length === 2);
 		const data = normalized.map((f) => f());
 		assert.deepEqual(data, [2, 1]);
 	});
 
-	it("normalizeFlexListEager", () => {
+	it("correctly normalizes lists to be eager", () => {
 		const list = [2, (): 1 => 1] as const;
 		const normalized: readonly [2, 1] = normalizeFlexListEager(list);
 		assert.deepEqual(normalized, [2, 1]);
 	});
+
```

Comment:
you have a function to mark things and a way to check for marks. Might be nice to unit test those.

Reply:
Okay, done.